extensions/bombman/standalone.mjs

```diff
@@ -315,6 +325,74 @@ class Standalone {
   async s2s_provideStateFunc(srcExt, registerFunc) {
     this.helper.callS2sAPI(srcExt, registerFunc, this.helper.getListOfStateFunctions(this));
   }
+
+  /**
+   * kill player and tp
+   * @param {String} playerID
+   */
+  async killPlayer(playerID, endGame = false) {
+    const player = this.helper.gameState.getPlayer(playerID);
+    const target = player.mapCoord.copy();
+    target.x = 1;
+    target.y = 1;
+    this.helper.teleport(playerID, target, true);
+    this.playerIDs.delete(playerID);
+    if (this.playerIDs.size <= 1 && !endGame) {
+      await this.terminateGame();
+    }
+  }
+
+  /**
+   * @param {String} playerID playerID
+   */
+  teleportPlayer(playerID) {
+    const player = this.helper.gameState.getPlayer(playerID);
+    const target = player.mapCoord.copy();
+    target.x = 10;
+    target.y = 5;
+    this.helper.teleport(player.playerID, target, true);
+  }
+
+  async terminateGame() {
+    for (const playerID of this.playerIDs) {
+      console.log(playerID, 'Win');
+      // TODO: give item
+      // this.helper.callS2sAPI('item', 'giveItem', playerID, 'bombman_trophy', 1);
```

Comment:
Consider making the trophy item configurable through the configuration. If the configuration is not set, then we can skip.